composer.json

```diff
@@ -37,14 +37,14 @@
         "source": "https://github.com/MyIntervals/emogrifier"
     },
     "require": {
-        "php": "^5.5.0 || ~7.0.0 || ~7.1.0 || ~7.2.0",
+        "php": "^5.5.0 || ~7.0.0 || ~7.1.0 || ~7.2.0 || ~7.3.0",
         "ext-dom": "*",
         "ext-libxml": "*",
         "symfony/css-selector": "^3.4.0 || ^4.0.0"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.2.0",
-        "squizlabs/php_codesniffer": "^3.3.0",
+        "squizlabs/php_codesniffer": "^3.3.2",
```

Comment:
This is necessary, because 3.3.0 is not compatible with PHP >= 7.2 and we are ignoring platform requirements with PHP 7.3.